pci/graphic_drivers/nvidia-470xx-dkms/MHWDCONFIG

```diff
@@ -0,0 +1,57 @@
+# mhwd Driver Config
+
+NAME="video-nvidia-470xx-dkms"
+INFO="Closed source NVIDIA drivers for linux."
+VERSION="2022.01.30"
+FREEDRIVER="false"
+PRIORITY="15"
```

Comment:
not sure about this. Manajro is set to priority 4, which is lower than nvidia-390xx and it is a bit strange.